{
  "_annotation": {
    "cve_id": "CVE-2024-45846",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-45846.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:mindsdb:mindsdb:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "2981A174-DC5D-5773-A511-E16EE9C97BC6",
                "versionEndExcluding": "24.7.4.1",
                "versionStartIncluding": "23.10.3.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}